src/Goteo/Application/Lang.php

```diff
@@ -112,20 +112,11 @@ static public function trans($id, array $parameters = array(), $locale = null) {
         return static::$translator->trans($id, $parameters, null, $locale);
     }
 
-    /**
-     * Handy method for the transChoice() function of the Symfony Component Translator
-     */
-    static public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null) {
-        static::factory();
-        return static::$translator->transChoice($id, $number, $parameters, $domain, $locale);
-    }
```

Comment:
The method `Translator::transChoice()` got deprecated / removed. There's an alternative to use it, but since we aren't using our custom `transChoice()` method, I've thought about simply removing it.